scholia/app/templates/base.html

```diff
@@ -47,7 +47,8 @@
 
 <script type="text/javascript" src="{{ url_for('static', filename='jquery.dataTables.min.js') }}"></script>
 <script type="text/javascript" src="{{ url_for('static', filename='scholia.js') }}"></script>
-
+<link href="https://cdn.jsdelivr.net/npm/select2@4.1.0-rc.0/dist/css/select2.min.css" rel="stylesheet" />
```

Comment:
The thirdparty asset is a problem on Wikimedia servers.

Reply:
I add static files for select2